.ci/pipeline.yml

```diff
@@ -0,0 +1,31 @@
+resources:
+- name: rapidpro-docker
+  type: git
+  source:
+    uri: https://github.com/praekeltfoundation/rapidpro-docker
+    branch: master
+  webhook_token: prod-my-resource
+  check_every: 30m
+
+- name: rapidpro-image
+  type: docker-image
+  source:
+    repository: praekeltfoundation/rapidpro
+    username: {{rapidpro-dockerhub-username}}
+    password: {{rapidpro-dockerhub-password}}
+  check_every: 30m
```

Comment:
We should add a configurable `registry_mirror: {{registry-mirror}}` option to the `source` block for environments that need a registry mirror.